<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Group7</class>
 <widget class="QFrame" name="Group7">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OCR extension</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame {
	background: qlineargradient( x1:0 y1:0, x2:1 y2:0, stop:0 #4CA1AF, stop:1 #2C3E50);
}
QComboBox{
background-color:white;
border: none;
border-radius: 4px;
}
QRadioButton
{color:white;
}
QLabel
{color:white;
background:transparent;
}

QPushButton
{background:white;
color:black;
border-radius:15px;
}
QPushButton:hover
{color:#333;
border-radius:15px;
background:#49ebff;
}
QTextEdit
{background:rgb(255, 255, 255);
border-radius:5px;
}
QLineEdit
{background:rgba(255, 255, 255, 50);
border-radius:5px;
border:none;
color: white;
border-bottom:1px solid #e2e2e2;
}
QGroupBox
{color:white;
}
#groupvBox
{border: none;
}</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>240</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="rbtnAccu">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Accuracy</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtnSword">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Single Word</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelW">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>431</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Word Checker üìù</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSubmit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>140</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Run OCR,</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPath">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtResults">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>311</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelD">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>431</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Digit Checker üî¢</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSubmit_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>140</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Run OCR.</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="ocr.qrc"/>
 </resources>
 <connections/>
</ui>
